### PR TITLE
Added missing IrcCommands, Added # before channels, Updated to 4.7.1 Framework

### PR DESCRIPTION
**Pull request by SeeingBlue**
Added missing IrcCommands, Added # before channels, Updated to 4.7.1 Framework

### CODE REVIEW

File: TwitchChatSharp/IrcMessage.cs [hunk 1]
```diff
@@ -243,30 +247,31 @@ public static IrcMessage FromRawMessage(string raw)
...
                 }
                 while (i < raw.Length && raw[i] != ' ')
                     ++i;
             }
-            lens[(int) state] = raw.Length - starts[(int) state];
-            string cmd = raw.Substring(starts[(int) parserState.STATE_COMMAND], lens[(int) parserState.STATE_COMMAND]);
-
+            lens[(int)state] = raw.Length - starts[(int)state];
+            string cmd = raw.Substring(starts[(int)parserState.STATE_COMMAND], lens[(int)parserState.STATE_COMMAND]);
+            //int zero = 0;
```
Comment (3ventic) [inline TwitchChatSharp/IrcMessage.cs]: remove these comments please

File: TwitchChatSharp.sln [hunk 2]
```diff
@@ -1,12 +1,10 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.24720.0
+# Visual Studio 15
+VisualStudioVersion = 15.0.28307.168
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TwitchChatSharp", "TwitchChatSharp\TwitchChatSharp.csproj", "{31EED20F-8D47-44D8-A223-7D177CB417F6}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MinimalisticExample", "MinimalisticExample\MinimalisticExample.csproj", "{782B7BDC-0BB0-42EB-8D5F-8DB0F0D4640A}"
```
Comment (3ventic) [inline TwitchChatSharp.sln]: why is this removed?

File: TwitchChatSharp/TwitchChatSharp.csproj [hunk 3]
```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>TwitchChatSharp</RootNamespace>
     <AssemblyName>TwitchChatSharp</AssemblyName>
-    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.7.1</TargetFrameworkVersion>
```
Comment (3ventic) [inline TwitchChatSharp/TwitchChatSharp.csproj:12]: long term the target should be a .net standard version, not framework, so it can be used with either core or framework, but I'm not going to ask you to make that change

**Review by 3ventic — CHANGES_REQUESTED** [review]

File: TwitchChatSharp.sln (hunk 2 above)
Comment (SeeingBlue) [reply TwitchChatSharp.sln]: I just don't have any use for it. I'm fairly new to this process, can you just accept the changes related to missing commands? Anyways, just trying to be helpful.

File: TwitchChatSharp/TwitchChatSharp.csproj (hunk 3 above)
Comment (SeeingBlue) [reply TwitchChatSharp/TwitchChatSharp.csproj:12]: I don't have core installed and just updated it to what my bot is using.

**Comment by SeeingBlue** [comment]
I re-added the example project, removed the comments and recommitted it to my fork. I am a bit confused about this PR though. Should I just remove it and do a new one or can this one be updated? Kinda new to this whole process.

**Comment by gibbed** [comment]
@SeeingBlue Adding more commits will update your PR. That's why people usually use branches for PRs instead of master.

**Comment by SeeingBlue** [comment]
Thanks @gibbed 
At least in this case it did exactly what I needed then.
